core/src/main/java/io/seata/core/protocol/Version.java

```diff
@@ -31,7 +31,7 @@
     /**
      * The constant CURRENT.
      */
-    public static final String CURRENT = "0.6.1";
+    public static final String CURRENT = "0.7.0";
```

Comment:
How about use the version configured in pom.xml  dynamicly?

Reply:
or the version in meta-info in the jar ?

Reply:
But when we run with IDE, there is no meta-info here.

Reply:
> But when we run with IDE, there is no meta-info here.

I think it is a question , we can retain this issue temporarily and merge this pr